python_tests/artifact/test_backend.py

```diff
@@ -122,6 +128,10 @@ def test_study_artifact_not_found() -> None:
         assert status == 404
 
 
+@pytest.mark.skipif(
+    version.parse(optuna.__version__) < version.parse("3.4.0"),
+    reason="upload_artiract needs storage",
+)
```

Comment:
In my understanding, this decorator will be ignored due to the module level `pytest.skip` call in L12. Is that correct?

Reply:
```
if version.parse(optuna.__version__) < version.parse("3.3.0"):
    pytest.skip("optuna.artiracts module is introduced at v3.3.0", allow_module_level=True)
```

At the module level, we will skip tests < v3.3.0. But, only this test does not pass at v3.3.0. So, I added skipif only for this test.

Reply:
Ah, I understand. Thank you for your reply :pray: